src/scripts/zoom-controls.js

```diff
@@ -0,0 +1,206 @@
+import { FOV_PANORAMA, FOV_SPHERE } from "@services/constants";
+
+/** 
+ * Zoom controls for ThreeJS based on OrbitControls.js
+ * @see https://github.com/mrdoob/three.js/blob/r101/examples/jsm/controls/OrbitControls.js 
+ */
+export default class ZoomControls extends H5P.EventDispatcher {
+
```

Comment:
Something like this maybe?

```suggestion

/** @constant {number} ZOOM_MIN Minimum zoom value. */
  static ZOOM_MIN = 1;

/** @constant {number} ZOOM_MAX Maximum zoom value. */
  static ZOOM_MAX = 100;

```